doc/api_package.rst

```diff
@@ -132,7 +138,7 @@
 
   .. attribute:: reponame
 
-    Id of repository the package was installed from (string).
+    Id of repository (string).
```

Comment:
```suggestion
    Id of repository the package belongs to (@System for installed packages of unknown origin) (string).
```
Based on the code, the description of `reponame` should match the "otherwise" part of the description of `from_repo`.